<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="CustomLabel" name="labelOne">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomLabel" name="labelTwo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomLabel" name="labelThree">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="InfoLabel" name="infoOne">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="InfoLabel" name="infoTwo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="InfoLabel" name="infoThree">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>customLabel.h</header>
  </customwidget>
  <customwidget>
   <class>InfoLabel</class>
   <extends>QLabel</extends>
   <header>infoLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
